src/index.ts

```diff
@@ -0,0 +1,226 @@
+import { TxData, TypedTransaction } from '@ethereumjs/tx';
+import {
+  arrToBufArr,
+  bufferToHex,
+  ecsign,
+  isValidPrivate,
+  privateToPublic,
+  publicToAddress,
+  stripHexPrefix,
+  toBuffer,
+} from '@ethereumjs/util';
+import {
+  concatSig,
+  decrypt,
+  getEncryptionPublicKey,
+  normalize,
+  personalSign,
+  signTypedData,
+  SignTypedDataVersion,
+} from '@metamask/eth-sig-util';
+import { add0x, Eip1024EncryptedData, Hex, Keyring } from '@metamask/utils';
+import { keccak256 } from 'ethereum-cryptography/keccak';
+import randombytes from 'randombytes';
+
+const _TYPE = 'Simple Key Pair';
```

Comment:
```suggestion
export const SimpleKeyringType = 'Simple Key Pair';
```

Can you export this value? I think it's better to import it into the `KeyringController`, right it is a hardcode value